Cargo.toml

```diff
@@ -24,3 +24,6 @@ rustsec = "0.11"
 term = "0.5"
 serde_json = "1"
 
+[dev-dependencies]
+assert_cmd = "0.11"
+tempdir = "0.3.7"
```

Comment:
`tempdir` is deprecated (use `tempfile`)

Reply:
Fixed.